libs/libc/signal/sig_addset.c

```diff
@@ -62,13 +62,12 @@
  *
  ****************************************************************************/
 
-int sigaddset(FAR sigset_t *set, int signo)
+int nxsig_addset(FAR sigset_t *set, int signo)
 {
   /* Verify the signal */
 
   if (!GOOD_SIGNO(signo))
     {
-      set_errno(EINVAL);
       return ERROR;
```

Comment:
```suggestion
      return -EINVAL;
```
There are probably Return Value comments above that need to match.

Reply:
Done.